### PR TITLE
[2216] setup staging foreign tables

### PR DESCRIPTION
**Pull request by babebe**
## Summary
Fixes #{[2216](https://github.com/HHS/simpler-grants-gov/issues/2216)}

## Changes proposed
Mixin, Staging and Foreign attachment models created 
Migration file created

## Context for reviewers
Run migration. new table should be created in staging schema

### CODE REVIEW

File: api/src/db/models/foreign/attachement.py
Comment (chouinar) [inline api/src/db/models/foreign/attachement.py:1]: Typo in file name, should be `attachment.py`

File: api/src/db/models/foreign/attachement.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# match by oracle_fdw, but we are matching them for maintainability.
+#
+
+from src.db.models.legacy_mixin import synopsis_mixin
+
+from . import foreignbase
+
+
+class TsynopsisAttachment(foreignbase.ForeignBase, synopsis_mixin.TsynopsisAttachmentMixin):
+    __tablename__ = "tsynopsis_attachment"
```
Comment (chouinar) [inline api/src/db/models/foreign/attachement.py:14]: This needs to be exactly `tsynopsisattachment` - we keep the names 100% matching how they are in Oracle for simplicity, even when it isn't what we might want.

File: api/src/db/models/staging/attachment.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from src.db.models.legacy_mixin import synopsis_mixin
+from src.db.models.staging.staging_base import StagingBase, StagingParamMixin
+
+
+class TsynopsisAttachment(StagingBase, synopsis_mixin.TsynopsisAttachmentMixin, StagingParamMixin):
+    __tablename__ = "tsynopsis_attachment"
```
Comment (chouinar) [inline api/src/db/models/staging/attachment.py:6]: Also needs no underscore
```suggestion
    __tablename__ = "tsynopsisattachment"
```

Comment (babebe) [reply api/src/db/models/staging/attachment.py:6]: Fixed

File: api/src/db/models/foreign/attachement.py (hunk 1 above)
Comment (babebe) [reply api/src/db/models/foreign/attachement.py:14]: Fixed

File: api/src/db/models/foreign/attachement.py
Comment (babebe) [reply api/src/db/models/foreign/attachement.py:1]: fixed

File: api/src/db/models/staging/attachment.py (hunk 2 above)
Comment (chouinar) [reply api/src/db/models/staging/attachment.py:6]: Still needs to be fixed here

**Review by chouinar — APPROVED** [review]
LGTM!

**Pull request merged**